documentation/IDTA-01002-3/modules/ROOT/pages/Annex/IDTA-01002_ValueOnlySerializationExample.adoc

```diff
@@ -0,0 +1,152 @@
+[appendix]
+== ValueOnly-Serialization Example 
```

Comment:
This file is already in the https://github.com/admin-shell-io/aas-specs/tree/BiBo/IDTA-01001-3-1_MovePart2Sections branch of the was-specs repo. No need to add it here.

Reply:
removed